src/malli/generator.cljc

```diff
@@ -456,6 +456,26 @@
                       (-> (-min-max schema options)
                           (update :min #(some-> % double))
                           (update :max #(some-> % double))))))
+(defmethod -schema-generator :float [schema options]
+  (let [max-float #?(:clj Float/MAX_VALUE :cljs (.-MAX_VALUE js/Number))
+        min-float (- max-float)
+        props (m/properties schema options)
+        min-max-props (-min-max schema options)
+        min-max-clamped #?(:clj {:min (if-not (nil? (:min min-max-props))
+                                        (:min min-max-props)
+                                        min-float)
+                                 :max (if-not (nil? (:max min-max-props))
+                                        (:max min-max-props)
+                                        max-float)}
+                           :cljs min-max-props)
+        infinite? #?(:clj false :cljs (get props :gen/infinite? false))]
+    (->> (merge {:infinite? infinite?
+                 :NaN? (get props :gen/NaN? false)}
+                (-> min-max-clamped
+                    (update :min #(some-> % float))
+                    (update :max #(some-> % float))))
```

Comment:
I think `min-max-clamped` can be replaced with something like this:

```clojure
#(or (some-> % float) max-float)
```

Reply:
I couldn't quite replace the code exactly with your provided example, but I have refactored the code to remove the `min-max-clamped` with this:

```clojure
(-> min-max-props
  (update :min #(or (some-> % float)
                    #?(:clj min-float :cljs nil)))
  (update :max #(or (some-> % float)
                    #?(:clj max-float :cljs nil))))
```

This code is required to be this way because we do not want to default the boundaries in ClojureScript because we want to support generating Infinity.